modules/CuckooFilter/Cargo.toml

```diff
@@ -1,9 +1,11 @@
-[package]
-name = "CuckooFilter"
-version = "0.1.0"
-authors = ["u6940136@anu.edu.au <toastcraft.info@gmail.com>"]
-edition = "2018"
-
-# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
-
-[dependencies]
+[package]
+name = "CuckooFilter"
+version = "0.1.0"
+authors = ["u6940136@anu.edu.au <toastcraft.info@gmail.com>"]

```

Comment:
**[Required]** This list of authors needs to be updated with all the team members names who are working on the cuckoo filter